tests/lib/__init__.py

```diff
@@ -821,25 +821,26 @@ def _git_commit(
 
 
 def _vcs_add(script, version_pkg_path, vcs="git"):
-    all_vcs: Dict[str, Callable] = {"git": _git,
-                                "hg": _hg,
-                                "svn":_svn,
-                                "bazaar":_bazaar,
-                                }
-
-    if vcs in all_vcs:
-        # get corresponding vcs function and call it.
-        all_vcs[vcs](script, version_pkg_path)
+    if vcs == "git":
+        _git(script, version_pkg_path)
+    elif vcs == "hg":
+        _hg(script, version_pkg_path)
+    elif vcs == "svn":
+        _svn(script, version_pkg_path)
+    elif vcs == "bazaar":
+        _bazaar(script, version_pkg_path)
     else:
         raise ValueError(f"Unknown vcs: {vcs}")
 
     return version_pkg_path
 
+
 def _git(script, version_pkg_path):
```

Comment:
Let's also change these names to something like `_vcs_add_git`, to make it clearer that these are related to the `_vcs_add` function.

Reply:
we can, what do you think about `_add_git` instead?

Reply:
also, instead of  `_vcs_add`, I think `_add_vcs` sounds better. it sounds more verb-like haha